x/evm/keeper/statedb_test.go

```diff
@@ -792,3 +792,219 @@ func (suite *KeeperTestSuite) _TestForEachStorage() {
 		storage = types.Storage{}
 	}
 }
+
+func (suite *KeeperTestSuite) TestContext() {
+	// setup
+	suite.SetupTest()
+
+	amounts := []int64{
+		10000000000,
+		20000000000,
+		30000000000,
+	}
+
+	testCase := []struct {
+		name       string
+		initAmount int64
+		operation  func(stateDB *statedb.StateDB, from, to common.Address, amounts []int64)
+		/*
+			0: eth from before operation - eth from after operation
+			1: eth to after operation - eth to before operation
+			2: cosmos from before operation - cosmos from after operation
+			3: cosmos to after operation - cosmos to before operation
+		*/
+		expectedAmounts [4]int64
```

Comment:
Question:
Is this test case testing the balance of two accounts in which the amount is differ depending on the flush/revert call?